tests/test_util.py

```diff
@@ -1,7 +1,7 @@
 import datetime
 import os
 import unittest
-
+import  types
```

Comment:
follow PEP8 for import order https://peps.python.org/pep-0008/#imports

this should be 
import datetime
import os
import  types
import unittest